### PR TITLE
Improving the main copies in the landing page

### PR DESCRIPTION
**Pull request by jjmaestro**
Just improving (IMHO! ;) the English of some of the main copies in the landing page.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -300,7 +300,7 @@ <h3 class="text thin">If the buttons are important, so are the lists. Tuktuk giv
          <div class="row margin_bottom">
                 <div class="column_10 offset_1 text center">
                 <h1 class="text color theme">Forms & Tables</h1>
-                <h3 class="text thin">Get that forms & lists looks equally regardless of device and browser.</h3>
+                <h3 class="text thin">Make forms look gorgeous regardless of device and browser.</h3>
```
Comment (jjmaestro) [inline index.html:303]: I wasn't sure what you meant here... perhaps that forms look as ordered as lists? :-?  In any case I think my copy conveys the "forms look awesome" message :)

File: index.html [hunk 2]
```diff
@@ -244,7 +244,7 @@ <h3 class="text thin">If the buttons are important, so are the lists. Tuktuk giv
             </div>
 
             <div class="column_3">
-                <h4>Ordered Bullets</h4>
```
Comment (jjmaestro) [inline index.html:247]: Ordered lists have numbered bullets ;)

**Comment by jjmaestro** [comment]
I would also check the overall capitalization of Tuktuk: is it tuktuk, TukTuk, Tuktuk... :-?

**Comment by soyjavi** [comment]
THX!!!

**Pull request merged**